perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -22,6 +22,25 @@
 
 As other cloud providers deliver a managed MySQL service, we will add it here.
 
+To make this benchmark run for GCP:
+To run this benchmark for GCP if is required to install a non-default gcloud
```

Comment:
typo: "if"

specify reasons why it's required to install a non-default gcloud component (because ...), also specify the time/date of this requirement ("As of May 2017")